src/main/scala/org/broadinstitute/dsde/agora/server/business/AgoraBusiness.scala

```diff
@@ -42,40 +38,40 @@ class AgoraBusiness {
                  username: String): Option[AgoraEntity] = {
     val optEntity = AgoraDao.createAgoraDao(entityTypes).findSingle(namespace, name, snapshotId)
     optEntity match {
-      case Some(foundEntity) => Some(foundEntity.copy(url = Option(agoraUrl(foundEntity))))
+      case Some(foundEntity) =>
```

Comment:
Instead of returning an Option, we should just throw the error to be caught by Supervisor. Will also save some lines of code in QueryHandler